app/controllers/cohorts_controller.rb

```diff
@@ -56,8 +56,9 @@ def show
     params[:population] ||= 'Active Clients'
     load_cohort_names
     @cohort = cohort_scope.find(cohort_id)
-    # leave off the pagination here and return all the data
-    @cohort_clients = @cohort.search_clients(population: params[:population], user: current_user)
+    # Just fetch one client so the UI works, but we really are just using @cohort_clients in this context
+    # to determine if there is anyone on the page we're looking at
+    @cohort_clients = @cohort.search_clients(population: params[:population], user: current_user).limit(1)
```

Comment:
It looks to me like this will still fire the `preload` in `search_clients`, so it still does a bunch of queries. I don't think it is an issue, maybe note it?

Reply:
Yeah, shouldn't be an issue with just one client.  Gonna test, and can make a note if there's a problem.